conda/recipes/ucxx/meta.yaml

```
@@ -40,7 +40,12 @@ requirements:
     - ninja
```

Please copy these lines into `build` for `{{ compiler('cuda') }}`:

https://github.com/rapidsai/cudf/blob/de4a20ec9089a77e7c0a106794ba863e3d0083df/conda/recipes/libcudf/meta.yaml#L40-L45

And also these lines into `conda_build_config.yaml`:
https://github.com/rapidsai/cudf/blob/de4a20ec9089a77e7c0a106794ba863e3d0083df/conda/recipes/libcudf/conda_build_config.yaml#L7-L11

---

This suggestion includes the first piece

```suggestion
    {% if cuda_major == "11" %}
    - {{ compiler('cuda11') }} ={{ cuda_version }}
    {% else %}
    - {{ compiler('cuda') }}
    {% endif %}
    - cuda-version ={{ cuda_version }}
    - ninja
```

<hr>

The second is to add a `conda_build_config.yaml` file with this content adjacent to `meta.yaml`:

```yaml
cuda_compiler:
  - cuda-nvcc

cuda11_compiler:
  - nvcc
```